src/styles/pillar-themes.ts

```diff
@@ -0,0 +1,34 @@
+import { palette } from "@guardian/src-foundations";
+
+export interface PillarType {
+    colour?: string;
+    quote?: string;
+}
+
+export const pillarTheme = {
+    News: {
+        colour: palette.news.main,
+        quote:
+            "https://assets.guim.co.uk/images/email/icons/64855e3409f4927c771a5aca921997e4/quote-news.png"
+    },
+    Opinion: {
+        colour: palette.opinion.main,
+        quote:
+            "https://assets.guim.co.uk/images/email/icons/cc614106682d8de187a64eb222116f3a/quote-opinion.png"
+    },
+    Sport: {
+        colour: palette.sport.main,
+        quote:
+            "https://assets.guim.co.uk/images/email/icons/b4b9407f64d0305ff1cc9a9b95524411/quote-sport.png"
+    },
+    Arts: {
```

Comment:
Minor, but what motivated the capitalisation?

Reply:
The API returns the pillar in this format (with capitals and 'Arts' instead of 'Culture'), so we need to map these names to the ones used/expected by @src-foundations. The key in the object above is exactly how the pillar names are in the API.